### PR TITLE
feat: Modal support aria-* in closable

### PR DESCRIPTION
**Pull request by kiner-tang**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [X] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

- https://github.com/ant-design/ant-design/discussions/47188

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | - Modal support aria-* in closable <br /> - fix the issue in the demo where the p element covers the close button, resulting in invalid clicks <br/> - Support document.activeElement |
| 🇨🇳 Chinese | - 弹框支持在 closable 中配置 aria-* <br /> - 修复了演示中p元素覆盖关闭按钮导致无效点击的问题 <br /> - 支持 document.activeElement |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [X] Doc is updated/provided or not needed
- [X] Demo is updated/provided or not needed
- [X] TypeScript definition is updated/provided or not needed
- [X] Changelog is provided or not needed

### CODE REVIEW

**Comment by yoyo837** [comment]
@SocketSecurity ignore npm/esbuild@0.18.20
@SocketSecurity ignore npm/@swc/core@1.3.72
@SocketSecurity ignore npm/@fortawesome/fontawesome-common-types@6.5.1
@SocketSecurity ignore npm/@fortawesome/free-regular-svg-icons@6.5.1
@SocketSecurity ignore npm/@fortawesome/free-solid-svg-icons@6.5.1

**Comment by kiner-tang** [comment]
@SocketSecurity ignore npm/esbuild@0.18.20
@SocketSecurity ignore npm/@swc/core@1.3.72
@SocketSecurity ignore npm/@fortawesome/fontawesome-common-types@6.5.1
@SocketSecurity ignore npm/@fortawesome/free-regular-svg-icons@6.5.1
@SocketSecurity ignore npm/@fortawesome/free-solid-svg-icons@6.5.1

**Review by li-jia-nan — APPROVED** [review]

**Review by zombieJ — APPROVED** [review]

File: components/image/__tests__/__snapshots__/index.test.tsx.snap [hunk 1]
```diff
@@ -254,19 +254,24 @@ exports[`Image Default Group preview props 1`] = `
             tabindex="0"
           />
           <div
-            class="ant-image-preview-content"
+            style="outline: none;"
```
Comment (Sallyfafafa) [inline components/image/__tests__/__snapshots__/index.test.tsx.snap:257]: 为什么这里要把 content 往里挪一层

File: package.json [hunk 2]
```diff
@@ -132,11 +132,11 @@
     "rc-cascader": "~3.21.2",
     "rc-checkbox": "~3.1.0",
     "rc-collapse": "~3.7.2",
-    "rc-dialog": "~9.3.4",
+    "rc-dialog": "~9.4.0",
```
Comment (yoyo837) [inline package.json:135]: @wahiche-ideal This is what you are looking for.

**Pull request merged**